### PR TITLE
Article History

### PR DESCRIPTION
**Pull request by robertu7**
* add `/[name]/[mediaHash]/history` for viewing article versions;
* move edit article page to `/[name]/[mediaHash]/edit`;
* loosen edit article and allow to add version description;
* refactor edit page and remove `Article.drafts` queries (will be deprecate on backend side);
* remove the use of `useEditArticleDetailSupportSetting` on editing an article;

### CODE REVIEW

File: lang/zh-Hans.json [hunk 1]
```diff
@@ -756,6 +777,9 @@
   "Eb1U+/": {
     "defaultMessage": "储值成功"
   },
+  "EcglP9": {
+    "defaultMessage": "Key"
```
Comment (pitb2022) [inline lang/zh-Hans.json:781]: en?

File: lang/zh-Hans.json [hunk 2]
```diff
@@ -1029,6 +1057,10 @@
     "defaultMessage": "目前总营收",
     "description": "src/views/Circle/Analytics/IncomeAnalytics/index.tsx"
   },
+  "L5guA9": {
+    "defaultMessage": "Archived article (Author),",
```
Comment (pitb2022) [inline lang/zh-Hans.json:1061]: en?

File: lang/zh-Hans.json [hunk 3]
```diff
@@ -1209,6 +1244,9 @@
   "OpeFTV": {
     "defaultMessage": "输入六位数字交易密码"
   },
+  "Oru6G2": {
+    "defaultMessage": "The Fingerprint from IPFS, you can read it via a gateway"
```
Comment (pitb2022) [inline lang/zh-Hans.json:1248]: en?

File: lang/zh-Hant.json [hunk 4]
```diff
@@ -1029,6 +1057,10 @@
     "defaultMessage": "目前總營收",
     "description": "src/views/Circle/Analytics/IncomeAnalytics/index.tsx"
   },
+  "L5guA9": {
+    "defaultMessage": "Archived article (Author),",
```
Comment (pitb2022) [inline lang/zh-Hant.json:1061]: en?

File: lang/zh-Hant.json [hunk 5]
```diff
@@ -1209,6 +1244,9 @@
   "OpeFTV": {
     "defaultMessage": "輸入六位數字交易密碼"
   },
+  "Oru6G2": {
+    "defaultMessage": "The Fingerprint from IPFS, you can read it via a gateway"
```
Comment (pitb2022) [inline lang/zh-Hant.json:1248]: en?

File: src/views/ArticleDetail/Edit/PublishState/PendingState.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { Layout, Translate } from '~/components'
+
+const PendingState = () => {
+  return (
+    <Layout.Notice
+      color="green"
+      content={<Translate id="publishing" />}
+      subDescription={
+        <Translate
```
Comment (pitb2022) [inline src/views/ArticleDetail/Edit/PublishState/PendingState.tsx:9]: can use intl.formatMessage?

File: src/views/ArticleDetail/History/InfoHeader/index.tsx [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+            hasDisplayName
+            hasAvatar
+            avatarSize="md"
+            textSize="xs"
+            textWeight="md"
+            spacing="xtight"
+          />
+
+          <section className={styles.viewIpfs}>
+            {/* TODO: toast for mobile */}
```
Comment (pitb2022) [inline src/views/ArticleDetail/History/InfoHeader/index.tsx:70]: 👀

File: src/views/ArticleDetail/MetaInfo/index.tsx [hunk 8]
```diff
@@ -45,32 +47,46 @@ const MetaInfo = ({
...
-        <span className={styles.edited}>
-          &nbsp;
-          <FormattedMessage
-            defaultMessage="published on"
-            id="1i2T4a"
-            description="src/views/ArticleDetail/MetaInfo/index.tsx"
-          />
+        <span>
+          {version?.createdAt ? (
+            <FormattedMessage defaultMessage=" published" id="twEps9" />
```
Comment (pitb2022) [inline src/views/ArticleDetail/MetaInfo/index.tsx:66]: Lack of space in Chinese, use \&nbsp;?

Comment (pitb2022) [reply src/views/ArticleDetail/MetaInfo/index.tsx:66]: https://www.notion.so/matterslab/d996af3c11ee4bdf9912984454c0397a?v=c5182f775b24473692a6c40ff81fa212&p=6ed64aa0c1cd4572bb543a31b1a0d4e1&pm=s

Comment (robertu7) [reply src/views/ArticleDetail/MetaInfo/index.tsx:66]: fixed in https://github.com/thematters/matters-web/pull/4175/commits/30e9613b1542c61b4390319bebf0cece3e21a497

**Pull request merged**